src/bci_build/package/gcc/README.md.j2

```diff
@@ -0,0 +1,95 @@
+# The GNU Compiler Collection Container Image
+
+The GNU Compiler Collection (GCC) is an optimizing compiler for various
+architectures and operating systems. It is the default compiler in the GNU
+project and most Linux distributions, including SUSE Linux Enterprise and
+openSUSE.
+
+
+## How to use the image
+
+### Compile an application with a `Dockerfile`
+
+Normally, you'd want to compile an application and distribute it as part of a
+custom container image. To do this, create a `Dockerfile` similar to the one
+below. The `Dockerfile` uses this image to build a custom container image,
+copies the sources to a working directory, and compiles the application:
+{%- set reference_no_release = (image.reference | replace("-%RELEASE%", "") ) %}
+
+```Dockerfile
+FROM {{ reference_no_release }}
+WORKDIR /src/
+COPY . /src/
+RUN gcc main.c src1.c src2.c
+CMD ["./a.out"]
+```
+
+It is also possible to compile a static binary with gcc as part of a multistage
+build:
+
+```Dockerfile
+FROM {{ reference_no_release }} as builder
+WORKDIR /src/
+COPY . /src/
+RUN gcc -static main.c src1.c src2.c
+
+FROM registry.suse.com/bci/bci-micro:latest
+WORKDIR /build/
+COPY --from=builder /src/a.out /build/app
+CMD ["/build/app"]
+```
+
+Note that you must build a static binary to deploy it into bci-micro; otherwise
+shared libraries might be missing. You cannot deploy such an app into a
+`scratch` image, as it is not possible to statically link glibc.
+
+
+### Available build systems
+
+The container image comes with `make` by default. Other build systems and
+related utilities are available in the repository, and they can be installed
+using `zypper`. This includes the following:
+- `meson`
+- `cmake`
+- `ninja`
+- `autoconf` & `automake`
```

Comment:
usually that's called autotools (it includes libtool)

Reply:
autotools is not a valid package name, that's why I listed autoconf & automake explicitly